app/models/concerns/fog_extensions/debug/debug_http_instrumentor.rb

```diff
@@ -0,0 +1,67 @@
+module FogExtensions
```

Comment:
Scrap that, if you take my AS notifications advice, since it would probably then go into an initialiser.